src/js/Controllers/NavController.js

```diff
@@ -73,6 +75,13 @@ class NavController {
                 this.audioStreamUrl = rpc.params.url;
                 return {"rpc": RpcFactory.StartAudioStreamSuccess(rpc.id)};
             case "SetVideoConfig":
+                this.videoConfig = rpc.params.config;
+
+                if (this.webmSupport === undefined) {
+                    var video = document.getElementById('navi_stream');
+                    this.webmSupport = 'probably' === video.canPlayType('video/webm; codecs="vp8"');
```

Comment:
This returns 'probably'?

Reply:
probably, maybe, or an empty string if it knows it does not support